buildbot/buildbot/config_default.py

```diff
@@ -34,14 +34,19 @@
 CONFIG_OPTIONS = {
     'skipseashell': bool,
     'estimate': bool,
+    'directives': str,
     'make': bool,
+    'kernel': str,
```

Comment:
Maybe "name" would be a better name for this field than "kernel"? It's just some identifying string meant to be helpful to humans, without any prescribed deeper meaning than that…